app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/ImportationFilesManager.kt

```diff
@@ -92,12 +92,11 @@ class ImportationFilesManager @Inject constructor(
     suspend fun continuouslyCopyPickedFilesToLocalStorage() {
         filesToImportChannel.consume { fileToImport ->
             SentryLog.i(TAG, "Importing ${fileToImport.uri}")
-            val copiedFile = copyUriDataLocally(fileToImport.uri, fileToImport.fileName)
-
-            if (copiedFile == null) {
-                reportFailedImportation(fileToImport)
-                return@consume
-            }
+            val copiedFile = Dispatchers.IO {
```

Comment:
Wouldn't we want to use the `@IoDispatcher` dispatcher we always inject everywhere?

```suggestion
            val copiedFile = ioDispatcher {
```

Reply:
Done.